### ISSUE TITLE
Node labels appear when calling geom_tiplab twice

### ISSUE THREAD

**Issue by ac-simpson** [YuLab-SMU/ggtree#391]
Hi,
I am trying to make the font of just one tip label 'bold', while also displaying the node labels. I followed some examples I found online and tried calling geom_tiplab twice, once subsetting to just the tip label I want to be bold, and then subsetting to display the rest of the labels.

This works just fine, but when I call geom_tiplab twice, the node labels appear whether I want them to or not. In the code below, I would expect to see the node labels appear offset and blue, but I am also seeing them centered and in black font (i.e. they appear twice). Am I subsetting incorrectly? Thanks!

To sum up: what I want are just the blue node labels, not the black ones. (and also to have one of my tip labels bolded). Here is an example:
```
set.seed(13)
tree <- rtree(10)
tree <- makeNodeLabel(tree)
ggtree(tree) + 
    geom_tiplab(aes(subset=node<2),fontface="bold") + 
    geom_tiplab(aes(subset=node>1)) +
    geom_nodelab(color="blue", nudge_y=1)
```
![image](https://user-images.githubusercontent.com/26659916/114474184-88650700-9baa-11eb-8239-5cb754304d40.png)

**Comment by brj1** [comment]
You are correct in that you are subsetting incorrectly. The subset param of `geom_tiplab` is applied over all labels (not just tips). The default subset for `geom_tiplab` is `geom_tiplab = isTip`  which will disaply labels for tips only. In your case you just need to filter out the node labels (i.e. add ` && isTip`).

```R
set.seed(13)
tree <- rtree(10)
tree <- makeNodeLabel(tree)
ggtree(tree) + 
    geom_tiplab(aes(subset=node<2 && isTip),fontface="bold") + 
    geom_tiplab(aes(subset=node>1 && isTip)) +
    geom_nodelab(color="blue", nudge_y=1)
```

**Comment by ac-simpson** [comment]
Hi brj1, thanks for responding. However, when I run the code you supplied, it doesn't remove the second set of node labels (it makes them bold). Are you getting something different?

![image](https://user-images.githubusercontent.com/26659916/114479490-80f72b00-9bb5-11eb-872e-bf3426477727.png)

**Comment by brj1** [comment]
Oops I was only supposed to put one `&`. The following will work:

```R
set.seed(13)
tree <- rtree(10)
tree <- makeNodeLabel(tree)
ggtree(tree) + 
    geom_tiplab(aes(subset=node<2 & isTip),fontface="bold") + 
    geom_tiplab(aes(subset=node>1 & isTip)) +
    geom_nodelab(color="blue", nudge_y=1)
```

**Comment by ac-simpson** [comment]
Awesome, thanks a lot!
